.github/ISSUE_TEMPLATE/new_component.yaml

```diff
@@ -26,8 +26,9 @@ body:
       label: Is this a vendor-specific component?
```

Comment:
I wonder if there's a way to clarify this requirement. We've seen a few times now where this question isn't clear to new contributors.

Reply:
Definitely.